articles/index.md

```diff
@@ -44,8 +41,8 @@ The Quantum Development Kit preview provides a complete development and simulati
 | Resource Estimator | The resource estimator estimates the resources required to run a given instance of a Q# operation on a quantum computer. |
 | Visual Studio extension | This extension contains templates for Q# files and projects as well as syntax highlighting and IntelliSense support. |
 | Visual Studio Code extension | This extension contains syntax highlighting, as well as code snippets for Q# and IntelliSense support. |
-| `qsharp` for Python | The `qsharp` package for Python makes it easy to simulate Q# operations and functions from within Python.  
-
+| `qsharp` for Python | The `qsharp` package for Python makes it easy to simulate Q# operations and functions from within Python. |
+|  IQ# | IQ# (pronounced i-q-sharp) is an extension to the .NET Core SDK that provides the core functionality for compiling and simulating Q# operations. |
```

Comment:
IQ# provides the backend needed by Python and Jupyter Notebooks, but isn't used in the C# codegen usecase; it might be worth clarifying that this component is mainly used by features that work with the Jupyter ecosystem?

Reply:
I updated